elements-metadata.json

```diff
@@ -677,5 +677,17 @@
     "vendor_version" : null,
     "key" : "acimodelbank",
     "extended" : false
+  },
+  "Palette" : {
+    "image" : "https://images.cloudelements.io/Screen Shot 2019-02-12 at 10.34.33 AM201902121735.png",
+    "path" : "httppalette.se/element.json",
+    "hub" : "general",
+    "author" : "GuyVanWert",
+    "name" : "Palette",
+    "active" : true,
+    "description" : "Imported http://palette.se/ SOAP Element",
+    "vendor_version" : null,
+    "key" : "httppalette.se",
```

Comment:
@GuyVanWert Is this the element key you'd like to use? Would palette be a better option, you think?